plugins/UsersManager/lang/en.json

```diff
@@ -1,5 +1,10 @@
 {
     "UsersManager": {
+        "2FA": "2FA",
```

Comment:
I left it there separately in case TwoFactorAuth plugin is disabled. I don't think we load the translation then? We're not checking in the UI whether 2FA plugin is disabled in UsersManager as it's edge case and bit tricky. This way it still shows correct translation there.